### DISCUSSION TITLE
Peace TV Urdu

### DISCUSSION THREAD

**Discussion by pieshop6** [iptv-org/community#871] [Q&A]
### Channel ID

PeaceTVUrdu.ae

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/ae.m3u#L110-L111
